deploy/gk-deploy

```diff
@@ -152,10 +149,10 @@ check_pods() {
     pods=$(${CLI} get pod --no-headers --show-all --selector="${1}" 2>&1)
     if [[ "${prestatus}" != "${pods}" ]]; then
       debug "Checking status of pods matching '${1}':"
-      debug "$pods"
+      debug "-n" "$pods"
```

Comment:
No `-n` here, that would cause no newline to be printed after the debug output, which we want in this case.